### PR TITLE
Reduce logging noise

### PR DESCRIPTION
**Pull request by tuliren**
## What
- This PR intends to reduce the amount of periodic loggings from the server.
- See the screenshot below. Mosts of the information there is not useful:
  Screen Shot 2021-06-05 at 01 31 49
- ~Need some help to dig into why some of the changes do not work.~

## How
- Change periodic logging levels from info to debug.
- ~Change temporal logging level to debug (not working).~
- ~Change non-error logs to debug level in `RequestLogger` (not working).~

## Recommended reading order
- This is a short PR. Reviewing it only takes 30 seconds. No specific reading order is needed.

### CODE REVIEW

File: airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
     if (HttpStatus.isClientError(status) || HttpStatus.isServerError(status)) {
       LOGGER.error(logBuilder.toString());
     } else {
-      LOGGER.info(logBuilder.toString());
+      LOGGER.debug(logBuilder.toString());
```
Comment (tuliren) [inline airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java:107]: Somehow I am still seeing info level logs:

```
airbyte-server     | 2021-06-05 09:13:03 INFO i.a.s.RequestLogger(filter):107 - {workspace_app_root=/tmp/workspace/server/logs} - REQ 172.18.0.1 GET 200 /api/v1/health
```

File: temporal/dynamicconfig/development.yaml [hunk 2]
```diff
@@ -31,6 +31,9 @@ history.defaultWorkflowRetryPolicy:
       MaximumIntervalCoefficient: 100.0
       BackoffCoefficient: 2.0
       MaximumAttempts: 0
+log.level:
+  - value: "debug"
+    constraints: {}
```
Comment (tuliren) [inline temporal/dynamicconfig/development.yaml:36]: Still seeing info level logs:

```
airbyte-temporal   | {"level":"info","ts":"2021-06-05T09:13:32.886Z","msg":"none","service":"matching","component":"matching-engine","lifecycle":"Started","wf-task-queue-name":"/_sys/temporal-sys-batcher-taskqueue/1","wf-task-queue-type":"Workflow","logging-call-at":"matchingEngine.go:202"}
```

File: airbyte-commons/src/main/resources/log4j2.xml [hunk 3]
```diff
@@ -66,7 +66,7 @@
         <Logger name="com.leansoft.bigqueue" level="INFO" />
         <Logger name="io.netty" level="INFO" />
         <Logger name="io.grpc" level="INFO" />
-        <Logger name="io.temporal" level="INFO" />
+        <Logger name="io.temporal" level="WARN" />
```
Comment (tuliren) [inline airbyte-commons/src/main/resources/log4j2.xml:69]: Not working.

Comment (jrhizor) [reply airbyte-commons/src/main/resources/log4j2.xml:69]: Are you trying to reduce the logging in the temporal service container or temporal-related logging on other containers? This should only change the level for other containers.

File: airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobRetrier.java [hunk 4]
```diff
@@ -71,11 +71,17 @@ public void run() {
...
-    LOGGER.info("Job Retrier Summary. Incomplete jobs: {}, Job set to retry: {}, Jobs set to failed: {}",
-        incompleteJobs.size(),
-        failedJobs.get(),
-        retriedJobs.get());
+    LOGGER.debug("Completed Job Retrier...");
+
+    int incompleteJobCount = incompleteJobs.size();
+    int failedJobCount = failedJobs.get();
+    int retriedJobCount = retriedJobs.get();
+    if (incompleteJobCount > 0 || failedJobCount > 0 || retriedJobCount > 0) {
```
Comment (jrhizor) [inline airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobRetrier.java:79]: great idea

File: temporal/dynamicconfig/development.yaml (hunk 2 above)
Comment (jrhizor) [reply temporal/dynamicconfig/development.yaml:36]: Where did you see `log.level` as an option for the dynamic config? I don't see it in https://github.com/temporalio/temporal/blob/master/common/dynamicconfig/constants.go 

I assume there's something that can be provided to the container in the docker-compose file to change how this works.

Comment (jrhizor) [reply temporal/dynamicconfig/development.yaml:36]: FWIW I'm not that concerned about limiting internal temporal logging. It's noisy on the full view, but it's easy to just pull logs from the api and the scheduler and have a good view of Airbyte as a whole.

File: airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java (hunk 1 above)
Comment (jrhizor) [inline airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java:107]: I think it makes sense for our API server to log requests by default. We can do things like analyze our own logs for % successful requests / other analytics. It's also a good indicator that things are actually working and when it's receiving requests. This is something that I've actually run into while setting up production instances while debugging connectivity issues.

**Review by jrhizor — APPROVED** [review]
Approved % the `RequestLogger` change.

File: temporal/dynamicconfig/development.yaml (hunk 2 above)
Comment (tuliren) [reply temporal/dynamicconfig/development.yaml:36]: I was following this documentation about log configuration:
https://docs.temporal.io/docs/server/configuration/#log

Did not realize that the file should follow the go constant file.

Will revert the change.

File: airbyte-commons/src/main/resources/log4j2.xml (hunk 3 above)
Comment (tuliren) [reply airbyte-commons/src/main/resources/log4j2.xml:69]: I see. Make sense. I will revert this.

**Pull request merged**